src/CBLCollection.cc

```diff
@@ -68,9 +67,15 @@ namespace cbl_internal {
             CBLDocumentChange change = {};
             change.collection = _collection;
             change.docID = _docID;
+            
+            Retained<CBLDatabase> db;
             try {
-                _collection->database()->notify(this, change);
+                db = _collection->database();
             } catch (...) { }
```

Comment:
Worth logging here.  Silently swallowing errors tends to be bad.

Reply:
Added.